js/about/contributionStatement.js

```diff
@@ -160,10 +174,19 @@ class ContributionStatement extends ImmutableComponent {
   }
 
   get ContributionStatementHeader () {
+    const imgStyle = StyleSheet.create({
```

Comment:
curious: why separate the brave logo style here rather than keeping it with the other styles as before (`styles.braveLogo`)?